src/components/Incorporation/UploadRules.vue

```diff
@@ -74,9 +74,35 @@
             </li>
           </ul>
 
+          <p class="mt-4">
+            The Cooperative Association Regulation provide a
+            <a href="https://www.bclaws.gov.bc.ca/civix/document/id/complete/statreg/391_2000#ScheduleB">Schedule B</a>
```

Comment:
This would be a question for Severin but I think that having a link directly in the html can be vulnerable to xss attacks.

Reply:
Good question. I believe [XSS attacks](https://owasp.org/www-community/attacks/xss/) are when you specify actions (or v-html) in a template, so this is probably OK.

This is also a gov link so we probably trust it :)

Finally, I think our toolset looks for XSS vulnerabilities.

So I think this is fine.

Reply:
Alex, please resolve this conversation if OK with you.

Reply:
I know sonarcloud checks for security vulnerabilities in our other repos I don't think it checks the edit ui for some reason.

Reply:
Do you know in which repo SonarCloud checks for security vulnerabilities? I compared `.sonarcloud.properties` in Filings UI, Edit UI and Auth Web and I don't see any differences (other than an exclude statement).

Reply:
https://sonarcloud.io/organizations/bcgov-sonarcloud/projects it looks like SonarCloud checks lear, sbc-auth, business-filings-ui, sbc-pay, and sbc-common-componenets

Reply:
Hmm, I thought the local sonarcloud properties files would register each project with the site above, but I guess not. I do not have permissions to add projects to the site above.

@pwei1018 @rarmitag Can you help? We'd like to add business-create-ui and business-edit-ui (and any others you can think of) to the list of sonarcloud projects so that they get checked for security vulnerabilities.